packages/graphql-cli-validate-magento-pwa-queries/lib/index.js

```diff
@@ -67,7 +67,7 @@ async function validateQueries(context, argv) {
 
         // Validate our queries against that schema.
         context.spinner.start('Finding queries in files...');
-        const validator = getValidator({ clients, project });
+        const validator = getValidator({ clients, project, schemaPath });
```

Comment:
Now the validator will run the schema stored in the path rather than doing a "magical" search for the file within the directory the script is running. Basically we can run this from venia-concept but still check files in the sibling package, venia-ui.